test/e2e/specs/specs.go

```diff
@@ -1242,3 +1243,13 @@ func (t *TestPod) VerifyDefaultingFlagsArePassed(namespace string, expectedMachi
 	gomega.Expect(stdout).To(gomega.ContainSubstring(expectedMachineTypeFlagString),
 		"Should find MachineType flag string %q in stdout", expectedMachineTypeFlagString)
 }
+
+func (t *TestPod) VerifyProfileFlagsAreNotPassed(namespace string) {
+	stdout, stderr, err := e2ekubectl.RunKubectlWithFullOutput(namespace, "logs", t.pod.Name, "-c", "gke-gcsfuse-sidecar")
+	framework.ExpectNoError(err,
+		"Error accessing logs from pod %v, but failed with error message %q\nstdout: %s\nstderr: %s",
+		t.pod.Name, err, stdout, stderr)
+
+	gomega.Expect(stdout).To(gomega.Not(gomega.ContainSubstring("profile")),
+		"Should not find profile flag string %q in stdout")
```

Comment:
"Should not find profile flag string %q in stdout" -> "Should not find profile flag in stdout"